### PR TITLE
Add consumer-side tests for messaging.

### PR DESCRIPTION
**Pull request by OlgaMaciaszek**

### CODE REVIEW

File: src/main/java/com/example/sccshop/payment/PaymentResult.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+	String getUuid() {
+		return uuid;
+	}
+
+	boolean successful() {
+		return PaymentStatus.SUCCESS.equals(status);
+	}
+
+	enum PaymentStatus {
+		SUCCESS, FAILED
```
Comment (eddumelendez) [inline src/main/java/com/example/sccshop/payment/PaymentResult.java]: it should be `FAILURE` due to the changes in https://github.com/eddumelendez/scc-payment/pull/6

File: pom.xml [hunk 2]
```diff
@@ -58,6 +58,24 @@
 			<artifactId>spring-cloud-stream-test-support</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.spockframework</groupId>
+			<artifactId>spock-core</artifactId>
+			<version>1.1-groovy-2.4</version>
```
Comment (eddumelendez) [inline pom.xml]: move `spock` version to `properties` section in `pom.xml`

**Review by eddumelendez — CHANGES_REQUESTED** [review]

File: src/main/java/com/example/sccshop/payment/PaymentResult.java (hunk 1 above)
Comment (OlgaMaciaszek) [reply src/main/java/com/example/sccshop/payment/PaymentResult.java]: yes, I will merge it to the most recent master.

**Comment by OlgaMaciaszek** [comment]
@eddumelendez I have updated the PR. Have a look now please.

**Review by eddumelendez — APPROVED** [review]
LGTM

**Pull request merged**